tests/oss-fuzz/build.sh

```diff
@@ -36,21 +36,36 @@
 #     avif_fuzztest_enc_dec_incr@EncodeDecodeAvifFuzzTest.EncodeDecodeGridValid \
 #     --sanitizer address
 
-# Reset compile flags to build dav1d without fuzzer flags. The meson build system
-# is problematic with sanitizer flags.
+# Reset compile flags to build libyuv without fuzzer flags.
```

Comment:
If we need to build libyuv without fuzzer flags, add a commet to explain why.

Reply:
There was a good reason at some point: it simply did not build. It builds now with the common fuzzer flags so let's give it a try, thx for the PR.